Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
         // Use the label to assign the node to run the test.
         // It is recommended by SQUARE to not add the label
         docker {
-            image 'lsstts/develop-env:b76'
+            image 'lsstts/develop-env:develop'
```

Comment:
Uhm... is there a way to make it pull the image before starting the agent? This image needs to be updated before it is started.